src/components/Docs/Docs.tsx

```diff
@@ -0,0 +1,23 @@
+import { Link, Text } from '@chakra-ui/react';
+
+type DocsURLProps = {
+  url: string;
+  provider: string;
```

Comment:
This should be called `providerDisplayName` so it's obvious that the user of this component should put in the display name, not the regular name for the provider.